app/v2/services/AmendForeignPropertyPeriodSummaryService.scala

```diff
@@ -40,14 +40,14 @@ class AmendForeignPropertyPeriodSummaryService @Inject()(connector: AmendForeign
     correlationId: String): Future[ServiceOutcome[Unit]] = {
 
     val result = for {
-      ifsResponseWrapper <- EitherT(connector.amendForeignPropertyPeriodSummary(request)).leftMap(mapDownstreamErrors(ifsErrorMap))
+      ifsResponseWrapper <- EitherT(connector.amendForeignPropertyPeriodSummary(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))
```

Comment:
The for-comprehension isn't required. This could be refactored to 
`val result = EitherT(connector.amendForeignPropertyPeriodSummary(request)).leftMap(mapDownstreamErrors(downstreamErrorMap))`